.github/workflows/check-license.yml

```diff
@@ -0,0 +1,23 @@
+ This workflow will build a Java project with Maven
```

Comment:
This line is wrong in syntax. Should start with #

Reply:
Thanks for your review.